client/src/pages/components/dumbComponents.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
```

Comment:
General React architecture we'd want to have a file per component. Even if they're dumb.